app/api/router.py

```diff
@@ -11,3 +11,6 @@
 api_router.include_router(farms_router, prefix="/farms", tags=["farms"])
 api_router.include_router(fields_router, prefix="/fields", tags=["fields"])
 api_router.include_router(research_router, prefix="/research", tags=["research"])
+api_router.include_router(
+    drs_yield_router, prefix="/drs", tags=["drought_resistant_seed_yield"]
```

Comment:
Do we want this to be a little more descriptive? Maybe drought-resistant-seed? I know it's likely we only consume the API, but drs is very non descriptive.

Reply:
yeah I thought so but my brain was fried up by thinking names for fields and all the other stuff lol.... I will update it...

Reply:
No worries, thanks for taking a look.

Reply:
done...